src/core/LadspaControl.cpp

```diff
@@ -79,7 +79,7 @@ LadspaControl::LadspaControl( Model * _parent, port_desc_t * _port,
 				( m_port->max - m_port->min )
 				/ ( m_port->name.toUpper() == "GAIN"
 					&& m_port->max == 10.0f ? 4000.0f :
```

Comment:
This special case isn't handled. However, do we even need to keep this exception?